service.yaml

```diff
@@ -29,10 +33,20 @@ arguments:
     allowMajorVersions: true
     enablePrereleases: false
   reportingTeam: fnd-dtss
-  type:
-    - library
+  service: false
+  tutorial: false
+  vaultSecrets:
```

Comment:
Should this be leaked in public repo?

Reply:
they're just paths and jared needed them there to make something work. so idk